src/script/media/MediaConstraintsHandler.ts

```diff
@@ -57,28 +57,19 @@ export class MediaConstraintsHandler {
           DESKTOP_CAPTURER: {
             mandatory: {
               chromeMediaSource: 'desktop',
-              maxHeight: 1080,
-              minHeight: 1080,
             },
           },
           DISPLAY_MEDIA: {
             frameRate: 5,
```

Comment:
will this value be superceded in a 1:1 call?

Reply:
Yes, and in a conference call too.

By removing this value, the device determines the resolution. If you have a high resolution screen, this is it. WebRTC then reduces the resolution according to the bandwidth. This works well in 1:1 call. In Conference Call WebRTC can not reduces the resolution, currently.

Reply:
Usually not
In our test, the value was exceeded briefly, but that is probably due to the sending buffer. But the value hardly increased noticeably.